### PR TITLE
Census version

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
New version for census-rm,  we need our own version of this to alter.

# What has changed
Removal of Jenkins and Manifest files
renaming throughout code to census-rm-actionexporter
travis source image name changed
pom naming and github url

We have delayed any changes to fix pgcrypto for now as it should be on a separate ticket, and will required removing liquidbase history from db, or a new project

# How to test?
Build new image locally and push to your gcr

# Links
https://trello.com/c/6WZFG7cq/604-fork-actionexporter

### CODE REVIEW

File: src/main/resources/database/changes/release-16/address_primary_key_data_migration.sql [hunk 1]
```diff
@@ -4,3 +4,4 @@ UPDATE actionexporter.actionrequest AS ar
 SET addressFK = a.addressPK
 FROM actionexporter.address AS a
 WHERE ar.sampleunitref = a.sampleunitref;
+--WHAT IS THIS SCARY THING AND DO WE STILL NEED IT, I've never run it
```
Comment (liamtoozer) [inline src/main/resources/database/changes/release-16/address_primary_key_data_migration.sql]: @AdamHawtin was this a manual migration step which was run separately ?

File: .travis.yml [hunk 2]
```diff
@@ -34,4 +34,5 @@ cache:
 
 branches:
   only:
-    - rm-census
\ No newline at end of file
+    - master
+    - census-version
```
Comment (AdamHawtin) [inline .travis.yml]: This is going to push branch build images to our registry which will get deployed into our environments, I don't think we want that.

File: src/main/resources/database/changelog-master.yml [hunk 3]
```diff
@@ -45,5 +45,8 @@ databaseChangeLog:
   - include:
       file: database/changes/release-15/changelog.yml
 
+  - include:
+          file: database/changes/release-15/changelog.yml
```
Comment (AdamHawtin) [inline src/main/resources/database/changelog-master.yml]: Duplicates the 'include' above?

File: src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql [hunk 4]
```diff
@@ -1,4 +1,5 @@
 -- Create postgres extension to allow generation of v4 UUID
+
```
Comment (AdamHawtin) [inline src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql]: Best not to edit existing migrations without a good reason?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
A few comments, the badges on the README still point to the business/social version as well.

File: src/main/resources/database/changelog-master.yml (hunk 3 above)
Comment (AdeHarrison) [inline src/main/resources/database/changelog-master.yml]: typo?

File: src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql (hunk 4 above)
Comment (AdeHarrison) [inline src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql]: typo?

**Review by AdeHarrison — APPROVED** [review]
travis image push tested successfully and apart from 2 very, very minor comments below which do not prevent service starting, looks good to me

File: .travis.yml (hunk 2 above)
Comment (LukeLoze1) [reply .travis.yml]: Ade is just having a play with travis, he'll revert it

File: src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql (hunk 4 above)
Comment (LukeLoze1) [reply src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql]: yeap

Comment (LukeLoze1) [reply src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql]: removed

File: Dockerfile [hunk 5]
```diff
@@ -1,11 +1,11 @@
 FROM openjdk:8-jre-slim
 
 VOLUME /tmp
-ARG JAR_FILE=actionexportersvc-*.jar
+ARG JAR_FILE=census-rm-actionexportersvc-*.jar
 RUN apt-get update
 RUN apt-get -yq install curl
 RUN apt-get -yq clean
-COPY target/$JAR_FILE /opt/actionexportersvc.jar
```
Comment (rjweeks) [inline Dockerfile:8]: This step fails for me because there is more than one Jar file that matches the JAR_FILE pattern.

**Comment by rjweeks** [comment]
Built Docker image locally and ran the acceptance tests. All tests passed.

File: Dockerfile (hunk 5 above)
Comment (LukeLoze1) [reply Dockerfile:8]: Are you able to reproduce this, the pattern above seems to be repeated in other services, such as the case-svc?

File: .travis.yml [hunk 6]
```diff
@@ -21,7 +21,7 @@ install: mvn install -DskipTests -DskipITs -Ddocker.skip -Ddockerfile.skip -Dmav
 script: mvn verify cobertura:cobertura-integration-test
 
 after_success:
-- if [ "$TRAVIS_BRANCH" == "rm-census" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+- if ([ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]) || [ "$TRAVIS_BRANCH" == "census-version" ]; then
```
Comment (AdamHawtin) [inline .travis.yml]: `|| [ "$TRAVIS_BRANCH" == "census-version" ]` needs to be removed

File: .travis.yml (hunk 2 above)
Comment (AdamHawtin) [inline .travis.yml]: census-version should be removed

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**